### PR TITLE
fix(pie-docs): DSW-1604 replace fozzie colour tokens with pie-css tokens

### PR DESCRIPTION
**Pull request by raoufswe**
## Describe your changes (can list changeset entries if preferable)

- replace fozzie colour tokens with pie-css tokens

## Author Checklist (complete before requesting a review)
- [ ] I have performed a self-review of my code
- [ ] If it is a core feature, I have added thorough tests
- [ ] If it is a `PIE Docs` change, I have reviewed the Docs site preview
- [ ] If it is a component change, I have reviewed the Storybook preview
- [ ] If there are visual test updates, I have reviewed them properly before approving

### Reviewer 1
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### Reviewer 2
- [ ] If it is a `PIE Docs` change, I have reviewed the PR preview
- [ ] If there are visual test updates, I have reviewed them

### CODE REVIEW

File: apps/pie-docs/src/assets/styles/components/_toggle.scss [hunk 1]
```diff
@@ -1,30 +1,27 @@
...
 $toggle-padding: 2px;
 $toggle-item-height: 44px;
 $toggle-item-fw: f.$font-weight-regular;
 $toggle-item-fw--checked: f.$font-weight-bold;
 $toggle-item-padding-y: 10px;
 $toggle-item-padding-x: var(--dt-spacing-e);
-$toggle-item-bg: f.$color-container-strong;
+$toggle-item-bg: var(--dt-color-container-strong);
 
 // This is really annoying, but it's the only way to get darkness to work with Custom Props
```
Comment (xander-marjoram) [inline apps/pie-docs/src/assets/styles/components/_toggle.scss:14]: Looks like this comment can be deleted now?

File: .changeset/shy-onions-lick.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+"pie-docs": minor
+---
+
+[Changed] - Replaced fozzie colour tokesn with pie-css tokens
```
Comment (xander-marjoram) [inline .changeset/shy-onions-lick.md:5]:
```suggestion
[Changed] - Replaced fozzie colour tokens with pie-css tokens
```

**Review by kevinrodrigues — APPROVED** [review]

**Review by fernandofranca — APPROVED** [review]

File: apps/pie-docs/src/assets/styles/components/_toggle.scss (hunk 1 above)
Comment (raoufswe) [reply apps/pie-docs/src/assets/styles/components/_toggle.scss:14]: will remove it in my next PR 🤝

Comment (raoufswe) [reply apps/pie-docs/src/assets/styles/components/_toggle.scss:14]: never mind I need to update the changeset anyway

**Review by fernandofranca — APPROVED** [review]

**Pull request merged**